cypress/e2e/notebooks.cy.ts

```diff
@@ -0,0 +1,25 @@
+describe('Notebooks', () => {
+    beforeEach(() => {
+        cy.intercept('GET', /api\/projects\/\d+\/insights\/\?.*/).as('loadInsightList')
+        cy.intercept('PATCH', /api\/projects\/\d+\/insights\/\d+\/.*/).as('patchInsight')
```

Comment:
```suggestion
```

I don't think it matters but for tidiness I don't think we're using these

Reply:
I assumed they were needed as part of loading the insights page first but removed them for the sake of cleanliness